### ISSUE TITLE
Problems after creating derived collation and using it as default collation

### ISSUE THREAD

**Issue by ron90** [mariuz/flamerobin#260]
### Platform
**Flamerobin:** Version 0.9.3 - e6015349 (64 bit)
**OS:** Windows 10 - (64 bit)
**FireBird:** Version 4.0.1.2692 (64 bit) - This problem is also with FireBird 3.0.9

### Description
When a new (derived) collation is created from existing collation and it is set as a default collation for a character set then
FlameRobin has problems.

### Steps to reproduce
1) Create a database with default character set as **ISO8859_1**
![1](https://user-images.githubusercontent.com/65551754/174084661-0fa06e44-1e0b-4c88-ac1f-ff9176f3a327.png)

2) Creating a case-insensitive collation based on an already existing collation in the database
_CREATE COLLATION EN_US_CI_AI
FOR ISO8859_1
FROM EN_US
NO PAD 
CASE INSENSITIVE
ACCENT INSENSITIVE;_

3) Set created collation as default for ISO8859_1
_ALTER CHARACTER SET ISO8859_1 SET DEFAULT COLLATION EN_US_CI_AI;_

4) Create a table normally
_CREATE TABLE Contacts
(
    ContactPK INT not null,
    ContactName VARCHAR(20) not null
);_

### Problems
**Column Data Types are not displayed in Tree View, Properties page & DDL. Instead of column data type, RDB$1 / RDB$2 is displayed.**

![2](https://user-images.githubusercontent.com/65551754/174086113-17632357-3530-422c-a70c-6f0fb7bea591.PNG)

![3](https://user-images.githubusercontent.com/65551754/174086260-7166b9c8-ec8a-4745-990c-ad1eee3b5538.PNG)

![4](https://user-images.githubusercontent.com/65551754/174086304-84f30c32-ad37-4ae2-a0bc-23e80923d41f.PNG)

![5](https://user-images.githubusercontent.com/65551754/174086347-29cf69de-2761-42e6-b2d8-80e09c8a559b.png)

**Select query and Browse Data does not work on System Tables. Flamerobin hangs/crashes after selecting Yes or No on the error message box.**
 
![6](https://user-images.githubusercontent.com/65551754/174086791-ace89d55-9600-4544-a53b-b8753db98880.png)
![7](https://user-images.githubusercontent.com/65551754/174086841-cb7c3113-ab73-4441-97a4-b90fd2f40f76.png)

![8](https://user-images.githubusercontent.com/65551754/174086867-aa64db3b-76f2-4c9e-a1a4-3495ed27d46d.PNG)

### Workaround Solution
While connecting to database if **UTF8** is selected in character set then above problems are not found.

![9](https://user-images.githubusercontent.com/65551754/174087850-49c2f91e-9d21-49c3-8451-60cfc1b8f76a.png)

**Comment by arvanus** [comment]
I believe this is a Firebird bug, as IBExpert have a strange behavior too
Posted a thread at Firebird-support 
https://groups.google.com/g/firebird-support/c/0r4Platf5fI

**Comment by ron90** [comment]
Thankyou @arvanus and others for looking into this and taking it forward to the Firebird support team.

As per your reply, I also checked and found that `NO PAD` could be creating this problem. I am unaware of the internals so the developers and Firebird people are the best judge.

I tried creating the collation without specifying `NO PAD` clause as well as with `PAD SPACE` option. In both cases there are no issues, everything works ok.

**Comment by arvanus** [comment]
Please, try this snapshot https://github.com/mariuz/flamerobin/releases/tag/0.9.3.12
Ad close the issue if everything is OK

**Comment by ron90** [comment]
Hello, I tried with the new snapshot.

Still there are two issues I found:
1) The system tables query you mentioned does not return any rows yet. 
```
select * from rdb$fields f
join rdb$relation_fields r
  on f.rdb$field_name=r.rdb$field_source
where r.rdb$relation_name = 'CONTACTS';
```
No runtime error now in selecting from other system tables.

2) Field properties page now shows data type of the column in _Domain_  instead of _Datatype_.
![FieldProperties](https://user-images.githubusercontent.com/65551754/175873345-eb1933a6-b20b-4011-bd47-c6bf557cfbb0.png)
